dependabot-auto-merge/__tests__/autoMerge.ts

```diff
@@ -0,0 +1,354 @@
+import {
+	getMergeableDescriptionTypeA,
+	getUnmergeableDescriptionTypeA,
+	mergeableDescriptions,
+	unmergeableDescriptions,
+} from '../__fixtures__/autoMerge/autoMerge';
+
+import * as autoMerge from '../src/autoMerge';
+
+const {
+	isPullRequestApprovable,
+	isPullRequestMergeable,
+	isVersionBumpSafeToMerge,
+	markAutoMergePullRequest,
+} = autoMerge;
+
+import type { PartialDeep } from 'type-fest';
+import {
+	CheckRunDetails,
+	Octokit,
+	PullRequestFromGet,
+	PullRequestFromList,
+	PullRequestReview,
+} from '../src/types/GitHub';
+import { faker } from '@faker-js/faker';
+import { getPullRequests } from '../src/github';
+
+const mockGetPullRequest = (): PartialDeep< PullRequestFromGet > => {
```

Comment:
I copy-pasted the mocking code from `github.ts` to `autoMerge.ts`, and I don't think that's the best idea.

Does anyone have a recommendation on how we can keep this DRY? I've never been able to figure it out.